.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+ # For anything not explicitly taken by someone else:
+*  @anuraaga @bogdandrutu @jkwatson
```

Comment:
we should find some maintainers for this repo who care about having an independent proto artifact. :)

Reply:
Hoping this will be a pretty trivial repo after the initial commits :)